tuf/requests_fetcher.py

```diff
@@ -14,7 +14,7 @@
 import urllib3.exceptions
 
 from tuf import exceptions
-import tuf.settings
+from tuf import settings
```

Comment:
Nit pick: I think the patch in this file should have gone into 7841008a. (Note that I mostly point this out so that you see that I actually read the diff 😄 . Feel free to ignore.)